README.md

```diff
@@ -893,6 +895,72 @@ export function buildFluidImageData(
+
+#### createImgixGatsbyTypes
+
+This is the main function that should be used to reuse functionality from this library.
+
+##### API
+
+| Parameter          | Type                              | Required | Description                                                                                                                                                                                                                                                                       |
+| ------------------ | --------------------------------- | -------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| imgixClientOptions | `Object`                          |          | Any extra configuration to pass to new ImgixClient from @imgix/js-core (see [here](https://github.com/imgix/js-core#configuration) for more information).                                                                                                                         |
+| resolveUrl         | `(source) => string \| undefined` | ✔        | A callback to resolve the image URL from the source data provided to the resolver. The source data is provided by the resolver of the type you are adding the image fields too.                                                                                                   |
+| resolveWidth       | `(source) => number \| undefined` |          | This callback can be used to provide the source (uncropped) width of the target image from the source data, if known. The data passed to this callback is the same as is passed to `resolveUrl`.                                                                                  |
+| resolveHeight      | `(source) => number \| undefined` |          | This callback can be used to provide the source (uncropped) height of the target image from the source data, if known. The data passed to this callback is the same as is passed to `resolveUrl`.                                                                                 |
+| allowListFields    | `Array`                           |          | This can be used to restrict the types and fields generated to avoid creating extra unnecessary types. Possible values: url, fixed, fluid, gatsbyImageData.                                                                                                                       |
+| defaultParams      | `Object`                          |          | Default imgix parameters to apply for these images.                                                                                                                                                                                                                               |
```

Comment:
Is there the ability to provide default placeholder params? I see this is left out of the main plugin options so no worries if this is out of scope.

Setting default placeholder params could help define the quality and size of the placeholder, allowing plugin authors or, by extension, app developers, to tweak generated base64 placeholder data.

Reply:
I'll release this in a separate PR, thanks for the reminder!